src/camera.cpp

```diff
@@ -484,13 +484,12 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime,
 
 void Camera::updateViewingRange()
 {
+	f32 viewing_range = g_settings->getFloat("viewing_range");
+	m_draw_control.wanted_range = viewing_range;
 	if (m_draw_control.range_all) {
 		m_cameranode->setFarValue(100000.0);
 		return;
 	}
-
```

Comment:
This is do that view range get's updated even while fog is turned off. No perf detriment. Makes it nicer to change the view range to retrieve more (or less) data from the server, while seeing what's going on.